snippets/markdown/image/tensorflowjs/p5js.md

```diff
@@ -50,6 +50,8 @@ Open up the code snippet below directly in the [p5.js Web Editor](https://editor
   function classifyVideo() {
     flippedVideo = ml5.flipImage(video)
     classifier.classify(flippedVideo, gotResult);
+    flippedVideo.remove();
```

Comment:
Oh, thanks for catching this! @joeyklee I wonder if there is a more elegant way for us to handle this on the ml5 side?

Reply:
@AliKarpuzoglu - thank you for catching this. 
@shiffman - Most definitely. I can take a look at this.